pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImplTest.java

```diff
@@ -419,19 +425,41 @@ public boolean test(NamespaceBundle namespaceBundle) {
             assertTrue(ex.getMessage().contains("cannot be transfer to same broker"));
         }
     }
+
     @DataProvider(name = "isPersistentTopicTest")
     public Object[][] isPersistentTopicTest() {
-        return new Object[][] { { true }, { false }};
+        return new Object[][]{
+                {true, SubscriptionType.Exclusive},
```

Comment:
I think {SubscriptionType.Exclusive, SubscriptionType.Shared} type could be enough to cover.